Source/Addendum/IMAUtils.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  IMAUtils.swift
+//  PrebidMobile
+//
+//  Created by Punnaghai Puviarasu on 10/29/20.
+//  Copyright © 2020 AppNexus. All rights reserved.
+//
+
+import Foundation
+
+public final class IMAUtils: NSObject {
+    
+    @objc
+    public static let shared = IMAUtils()
+    
+    private override init() {}
+    
+    @objc
+    public func constructAdTagURLForIMAWithPrebidKeys (adUnitID:String, customKeywords: [String:String]) -> String{
+        let adServerURL = "https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&output=vast&unviewed_position_start=1&gdfp_req=1&env=vp"
```

Comment:
should not be fixed sz
either have the publisher pass in the size or use hb_size

Reply:
also android has changed to output=xml_vast4 based on @yoalex5 feedback